### PR TITLE
Role for forwarding server mail to SMTP server

### PR DESCRIPTION
**Pull request by jbzdak**
A role that sets up posix server that forwards all local mail to a designated email. 

Author's concerns: 
- If you have a shell account you can send emails from our server using the `mail` command. If bad people get accounts on our servers, this will be least of our concerns though.

### CODE REVIEW

**Comment by antoviaque** [comment]
> If you have a shell account you can send emails from our server using the mail command. If bad people get accounts on our servers, this will be least of our concerns though.

Yes, if they have shell access they can send emails anyway unless we completely block outbound ports - and they can definitely do much more harm than than.

File: defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+# defaults file for forward-mails-to-gmail
```
Comment (antoviaque) [inline defaults/main.yml]: To GMail? probably a leftover

**Comment by antoviaque** [comment]
:+1:

**Comment by jbzdak** [comment]
Squashed and merging

**Pull request merged**